pandas/plotting/_matplotlib/__init__.py

```diff
@@ -46,7 +46,7 @@
     register(explicit=False)
 
 
-def plot(data, kind, **kwargs):
```

Comment:
This doesn't feel like the right place for a fix. All other kwargs are passed through to the plotting class.

I'm also not entirely sure about a keyword controlling the dtype of data that gets plotted. I'm not sure what is best to do here.

Reply:
thanks @TomAugspurger what if I just add boolean into accepted type list?

Reply:
No, I think it needs to be an option somewhere, otherwise we're changing the plotting API. I'm just not sure where the option should go :/

Reply:
I submitted another version, probably not the most ideal, feel free to comment and would be super nice to let me know why it's not ideal enough, i am very interested in learning how to think to optimize large code base ^^ thanks! @TomAugspurger